website/components/FAQ.tsx

```
@@ -0,0 +1,86 @@
+                        </p>
+                    </div>
+
+                    {/* Right column — accordion */}
+                    <div className="glass-card rounded-2xl p-1">
+                        {faqs.map((faq, index) => (
+                            <div key={index} className={index > 0 ? 'border-t border-white/[0.04]' : ''}>
+                                <button
+                                    onClick={() => setOpenIndex(openIndex === index ? null : index)}
+                                    className="flex w-full items-center justify-between gap-4 px-6 py-5 text-left transition-colors hover:bg-white/[0.02]"
+                                >
+                                    <span className="text-[15px] font-medium text-[--color-text]">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)